.editorconfig

```diff
@@ -146,6 +146,5 @@ csharp_preserve_single_line_blocks = true
 dotnet_diagnostic.IDE0055.severity = error
 # Disable "IDE0130: Namespace does not match folder structure" because VS Code Cleanup applies it automatically and breaks things.
 dotnet_style_namespace_match_folder = false
-dotnet_diagnostic.IDE0058.severity = none
```

Comment:
Even though this explicitly shows the return(ed) value is ignored, I doubt it will add any value (like preventing a but due to ignored value)

Note that in golang this is even required by the compiler, so it is a (forced) common practice for me. I just have my doubts for the usefulness in C#/this project.

Reply:
Regarding this project, I'm also unsure.
But generally speaking, it helps refactoring. Just search for "_ =" and mark all these methods for refactoring. If their return value is not used, why return it in the first place?

Reply:
> why return it in the first place?

Because they are external-library methods. It's especially prevalent in the parts of code that use the SDL library. I'll take another look at the result to see if it really improves readability.

Reply:
Ah, ok. I didn't mean to get into specifics there, I was just speaking more generally.
3rd party methods are a different thing, yes.

Reply:
The point is that the most Discards are indeed used for the library method. That raises the question whether we need the Discards at all as we cannot fix the issue because they are 3rd-party methods. I'll think on that.

Reply:
> The point is that the most Discards are indeed used for the library methods

After looking at the code more, I'm not that sure if my previous statement is true. There are many usages of the YAFC methods that simply don't use the return values. 

> Even though this explicitly shows the return(ed) value is ignored, I doubt it will add any value (like preventing a bug due to ignored value)

If not by preventing bugs, it will add value by reducing the amount of effort required to read the code. It will indicate to a reader that it's not a bug that YAFC uses its own functions without using the returned values.

Reply:
So they could be indicators for (outdated) code that needs some fixing? I can live with that :smile: